ResearchKit/ActiveTasks/ORKRecorder.m

```diff
@@ -178,7 +178,7 @@ - (NSString *)recorderType {
 }
 
 - (NSString *)logName {
-    return [NSString stringWithFormat:@"%@_%@", [self recorderType],self.identifier];
+    return [NSString stringWithFormat:@"%@_%@_%@", [self recorderType], self.step.identifier, self.identifier];
```

Comment:
r=me. We can just pull it out of the URL as a filename and translate it to what Sage requires, or use as-is.